README.md

```diff
@@ -41,33 +41,56 @@ instead of
 
 ## Syntax
 
-_NOTE: this section is a bit in flux after recent changes. Please use the latest released-to-npm release for a stable syntax; we promise we will nail everything down before doing another release._
-
 ### Top-Level Constructs
+The following is a partial EBNF for the top-level constructs in Ecmarkdown.
+
+```
+Document = Paragraph , [ { "\n\n" , Paragraph  } ] ;
 
-An Ecmarkdown **document**, processed using the `ecmarkdown.document(string)` export, is composed of multiple pararaphs.
+Paragraph = OrderedList | UnorderedList | NonList ;
 
-A **paragraph** can be either a list or non-list paragraph.
+OrderedList = OrderedListItem , [ { "\n" , OrderedListItem | UnorderedListItem } ] ;
```

Comment:
In that flavour of EBNF (i.e., ISO 14977), curly braces denote *zero*-or-more, not one-or-more. So, e.g. `[ { Foo } ]` should be just `{ Foo }`.